src/Google/Ads/GoogleAds/Lib/V1/GoogleAdsClientBuilder.php

```diff
@@ -1,6 +1,6 @@
 <?php
 /*
- * Copyright 2018 Google LLC
+ * Copyright 2019 Google LLC
```

Comment:
We don't modify copyright of the existing files.

Reply:
Done.